docs/configuring-playbook-synapse-admin.md

```diff
@@ -24,11 +22,38 @@ After configuring the playbook, run the [installation](installing.md) command ag
 ansible-playbook -i inventory/hosts setup.yml --tags=setup-all,start
 ```
 
-
 ## Usage
 
 After installation, Synapse Admin will be accessible at: `https://matrix.DOMAIN/synapse-admin/`
 
 To use Synapse Admin, you need to have [registered at least one administrator account](registering-users.md) on your server.
 
 The Homeserver URL to use on Synapse Admin's login page is: `https://matrix.DOMAIN`
+
+### Sample configuration for running behind Traefik 2.0
+
+Below is a sample configuration for using this playbook with a [Traefik](https://traefik.io/) 2.0 reverse proxy.
+
+This an extension to Traefik config sample in [own-webserver-documentation](./configuring-playbook-own-webserver.md).
+
+```yaml
+# Don't bind any HTTP or federation port to the host
+# (Traefik will proxy directly into the containers)
+matrix_synapse_admin_container_http_host_bind_port: ""
+
+matrix_synapse_admin_container_extra_arguments:
+    # May be unnecessary depending on Traefik config, but can't hurt
+    - '--label "traefik.enable=true"'
+
+    # The Synapse Admin container will only receive traffic from this subdomain and path
+    - '--label "traefik.http.routers.matrix-synapse-admin.rule=(Host(`{{ matrix_server_fqn_matrix }}`) && Path(`{matrix_synapse_admin_public_endpoint}`))"'
```

Comment:
Maybe `{matrix_synapse_admin_public_endpoint}` needs to be `{{ matrix_synapse_admin_public_endpoint }}`?

Reply:
you're correct. changed it :)